runtime/onert/core/include/loader/CircleLoader.h

```diff
@@ -18,6 +18,7 @@
 #define __ONERT_LOADER_CIRCLE_LOADER_H__
 
 #include "ir/Graph.h"
+#include "ir/operation/RoPE.h"
```

Comment:
This `include` looks unnecessary.

Reply:
I misunderstood the header's call relationship again.
i think I overlooked calling `BaseLoader.h` -> `core\include\ir\Operations.Include.h` -> `ir/operation/RoPE.h` in CircleLoader.cc.
Thank you.

Reply:
Aha, Thank you for checking :)